docs/tutorials/scheduling.rst

```diff
@@ -353,7 +358,267 @@ and run the priority scheduler again.
+            # a different scheduler may consider which blocks have higher scores than others
+
+            # we want to start from the beginning and start scheduling
+            current_time = self.schedule.start_time
+            while current_time < self.schedule.end_time:
+                scheduled = False
+                i=0
+                while i < len(blocks) and scheduled is False:
+                    block = blocks[i]
+                    # the schedule starts with only 1 slot
+                    if len(self.schedule.slots) == 1:
+                        test_time = current_time
+                    # when a block is inserted, the number of slots increases
+                    else:
+                        # make a transition between the last scheduled block and this one
+                        transition = self.transitioner(schedule.observing_blocks[-1], block,
+                                                       current_time, self.observer)
+                        test_time = current_time + transition.duration
+                    # how far from the start is the time we are testing
+                    start_idx = int((test_time - self.schedule.start_time)/self.time_resolution)
+                    duration_idx = int(block.duration/self.time_resolution)
+                    # if any score during the block's duration would be 0, reject it
+                    if any(score_array[i][start_idx:start_idx+duration_idx] == 0):
+                        i +=1
+                    # if all of the scores are >0, accept and schedule it
+                    else:
+                        if len(self.schedule.slots) >1:
+                            self.schedule.insert_slot(current_time, transition)
+                        self.schedule.insert_slot(test_time, block)
+                        # advance the time, break this while loop and remove the block from the list
+                        current_time = test_time + block.duration
+                        scheduled = True
+                        blocks.remove(block)
+                # if every block failed, progress the time
+                if i == len(blocks):
+                    current_time += self.gap_time
+            return schedule
+
+Then to use our new scheduler, we just need to call it how we did
+up above::
+
+    >>> from astroplan.constraints import AtNightConstraint
+    >>> from astroplan.scheduling import Schedule, ObservingBlock
+    >>> from astroplan import FixedTarget, Observer, Transitioner
+    >>> from astropy.time import Time
+
+    >>> apo = Observer.at_site('apo')
+    >>> Deneb = FixedTarget.from_name('Deneb')
+    >>> M13 = FixedTarget.from_name('M13')
+    >>> blocks = [ObservingBlock(Deneb, 20*u.minute, 0)]
+    >>> blocks.append(ObservingBlock(M13, 20*u.minute, 0))
+
+    >>> # this will produce 0 second transitions
+    >>> transitioner = Transitioner()
```

Comment:
If we used a more realistic `Transitioner`, would that make the outputs more difficult to interpret? I think that's a bit too unrealistic.